boot/assets.go

```diff
@@ -63,3 +60,48 @@ func (o *TrustedAssetsInstallObserver) Seal() error {
 	// - initial seal
 	return nil
 }
+
+func NewTrustedAssetsUpdateObserver(model *asserts.Model) *TrustedAssetsUpdateObserver {
```

Comment:
(nitpick) This feels slightly iffy as NewXXX() usually creates the thing. Maybe `TrustedAssetsUpdateObserverForModel(model)` or something?

Reply:
yes, @mvo5 is right, I missed this in the other PR, but returning a nil from New is very unusual, this should probably return an error in the other case, with the doc comment explaining what's the only possible error case.

NewTrustedAssetsInstallObserver needs changing as well